pytest/tests/shardnet/scripts/create_account.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+account_id=$(cat /home/ubuntu/.near/shardnet/validator_key.json  | grep account_id | awk -F'"' '{print $4}')
```

Comment:
Useless use of cat. `grep account_id /home/ubuntu/.near/shardnet/validator_key.json`. Same on line 5.